### PR TITLE
Add import for json format(#65)

### LINKED ISSUE #65: arr-ai import file in other format (json, yml, sysl)
### Current arrai Version
v0.1.0 4add794

### Use-cases
import files in other format, i.e. `//./'config.yaml'`, `//./'app.sysl'`

### PR DESCRIPTION
**Pull request by tiagralee**
Fixes #65  .

Changes proposed in this pull request:
- syntax parse update grammar for local package to include file extension 
- change way to walk through the ast tree in local package

Checklist:
- [x] Added related tests
- [ ] Made corresponding changes to the documentation

### CODE REVIEW

File: syntax/package_test.go [hunk 1]
```diff
@@ -32,10 +32,18 @@ func TestPackageImport(t *testing.T) {
...
+func TestPackageImportWithQuote(t *testing.T) {
+	AssertCodesEvalToSameValue(t, `{1, 4, 9, 16}`, `//./'examples/simple/simple'`)
+}
+
 func TestPackageImportFromRoot(t *testing.T) {
 	AssertCodesEvalToSameValue(t, `{1, 4, 9, 16}`, `///examples/simple/simple`)
 }
 
+func TestJsonPackageImport(t *testing.T) {
+	AssertCodesEvalToSameValue(t, `{'location': 'Melbourne', 'name': 'foo'}`, `///examples/json/foo.json`)
```
Comment (cuminandpaprika) [inline syntax/package_test.go:44]:
```suggestion
	AssertCodesEvalToSameValue(t, `{'location': 'Melbourne', 'name': 'foo'}`, `//./examples/json/foo.json`)
```
Tests seem to be failing for me locally when I run make. I think you're missing a dot here?

Comment (tiagralee) [reply syntax/package_test.go:44]: Hi Cam, we have already add support for /// and //./ to be able to import from root, both should work. You can have a look at the line 35 to 41. 
btw, what is your test failing message?

Comment (tiagralee) [reply syntax/package_test.go:44]: https://github.com/arr-ai/arrai/pull/48 we already have the import from module root.

File: syntax/parse.go [hunk 2]
```diff
@@ -140,7 +145,7 @@ func (pc ParseContext) CompileExpr(b ast.Branch) rel.Expr {
 		"if", "call", "count", "touch", "get",
 		"rel", "set", "dict", "array", "embed", "op", "fn", "pkg", "tuple",
 		"xstr", "IDENT", "STR", "NUM",
-		"expr",
+		"expr", "PKG_DIR",
```
[the remark this answers is not in the document]
Comment (tiagralee) [reply syntax/parse.go]: Yes, this can be removed.

**Pull request merged**